web/sites/default/config/core.entity_view_display.node.guide.default.yml

```diff
@@ -31,21 +32,19 @@ third_party_settings:
     regions:
       content:
         - field_summary
-        - field_body
+        - field_body_content
```

Comment:
Since we are migrating away from the guide content type, you can leave the body field alone until we've finished migration.